hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -1258,10 +1279,22 @@ public ListKeysResult listKeys(String volumeName, String bucketName,
           break;
         }
       }
+      stopNano = Time.monotonicNowNanos();
     }
-
     boolean isTruncated = cacheKeyMap.size() > maxKeys;
-
+    if (perfMetrics != null) {
+      long averagePagination;
```

Comment:
```suggestion
      long keyCount;
```

Reply:
As per discussion, we decided to go with the name `averagePagination` for now. We may consider changing the same in a separate PR if needed, thanks! 

@muskan1012